src/postgres/src/test/regress/expected/yb_pg_copy2.out

```diff
@@ -0,0 +1,584 @@
+"Jackson, Sam","\\h"
+"It is \"perfect\".","	"
+"",
+COPY y TO stdout WITH CSV FORCE QUOTE *;
+"Jackson, Sam","\h"
+"It is ""perfect"".","	"
+"",
+-- Repeat above tests with new 9.0 option syntax
+COPY y TO stdout (FORMAT CSV);
+"Jackson, Sam",\h
+"It is ""perfect"".",
+"",
+COPY y TO stdout (FORMAT CSV, QUOTE '''', DELIMITER '|');
+Jackson, Sam|\h
+It is "perfect".|
+''|
+COPY y TO stdout (FORMAT CSV, FORCE_QUOTE (col2), ESCAPE E'\\');
+"Jackson, Sam","\\h"
+"It is \"perfect\".","	"
+"",
+COPY y TO stdout (FORMAT CSV, FORCE_QUOTE *);
+"Jackson, Sam","\h"
+"It is ""perfect"".","	"
+"",
+\copy y TO stdout (FORMAT CSV)
+"Jackson, Sam",\h
+"It is ""perfect"".",
+"",
+\copy y TO stdout (FORMAT CSV, QUOTE '''', DELIMITER '|')
+Jackson, Sam|\h
+It is "perfect".|
+''|
+\copy y TO stdout (FORMAT CSV, FORCE_QUOTE (col2), ESCAPE E'\\')
+"Jackson, Sam","\\h"
+"It is \"perfect\".","	"
+"",
+\copy y TO stdout (FORMAT CSV, FORCE_QUOTE *)
+"Jackson, Sam","\h"
+"It is ""perfect"".","	"
+"",
+--test that we read consecutive LFs properly
+CREATE TEMP TABLE testnl (a int, b text, c int);
+COPY testnl FROM stdin CSV;
+-- test end of copy marker
+CREATE TEMP TABLE testeoc (a text);
+COPY testeoc FROM stdin CSV;
+COPY testeoc TO stdout CSV;
+a\.
+\.b
+c\.d
+"\."
+-- test handling of nonstandard null marker that violates escaping rules
+CREATE TEMP TABLE testnull(a int, b text);
+INSERT INTO testnull VALUES (1, E'\\0'), (NULL, NULL);
+COPY testnull TO stdout WITH NULL AS E'\\0';
+1	\\0
+\0	\0
+COPY testnull FROM stdin WITH NULL AS E'\\0';
+SELECT * FROM testnull order by 1,2;
+ a  | b
+----+----
+  1 | \0
+ 42 | \0
+    |
+    |
+(4 rows)
+
+BEGIN;
+-- Like not supported. See: https://github.com/YugaByte/yugabyte-db/issues/1129
+CREATE TABLE vistest (a text);
+COPY vistest FROM stdin CSV;
+COMMIT;
+SELECT * FROM vistest order by 1;
+ a
+----
+ a0
+ b
+(2 rows)
+
+BEGIN;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+-- SAVEPOINT from original copy2.sql removed. See https://github.com/YugaByte/yugabyte-db/issues/1125.
+-- COPY FREEZE from original removed. See: https://github.com/yugabyte/yugabyte-db/issues/5156
+SELECT * FROM vistest order by 1;
+ a
+----
+ a1
+ b
+(2 rows)
+
+-- SAVEPOINT s1;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+SELECT * FROM vistest order by 1;
+ a
+----
+ d1
+ e
+(2 rows)
+
+COMMIT;
+SELECT * FROM vistest order by 1;
+ a
+----
+ d1
+ e
+(2 rows)
+
+BEGIN;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+SELECT * FROM vistest order by 1;
+ a
+----
+ a2
+ b
+(2 rows)
+
+-- SAVEPOINT s1;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV ;
+SELECT * FROM vistest order by 1;
+ a
+----
+ d2
+ e
+(2 rows)
+
+COMMIT;
+SELECT * FROM vistest order by 1;
+ a
+----
+ d2
+ e
+(2 rows)
+
+BEGIN;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+SELECT * FROM vistest order by 1;
+ a
+---
+ x
+ y
+(2 rows)
+
+COMMIT;
+BEGIN;
+TRUNCATE vistest;
+COPY vistest FROM stdin CSV;
+COMMIT;
```

Comment:
Why was `BEGIN` and `COMMIT` added?  Please add comments when you change things from the original test that are nonobvious.